src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -417,9 +417,10 @@ public string GenerateCMake(
             string[] dylibs = Directory.GetFiles(workspace, "*.dylib");
             if (targetRuntime == TargetRuntime.CoreCLR)
             {
-                foreach (string lib in dylibs)
+                string[] staticLibs = Directory.GetFiles(workspace, "*.a");
```

Comment:
A comment explaining why this is not using dylibs anymore and is using static libs might be good. It makes some amount of sense to me given the change further up in the diff around only doing the dynamic library on android, but the implications aren't obvious.

Also, should this be `-static.a` files only? Is blanket grabbing every `.a` file from the workspace what we want?

Reply:
Static linking is currently the default on iOS, but this may change in the future. There might be issues generating a static libraries for R2R. I suggest keeping this as a property and supporting both static and dynamic linking.

For static linking, we include the following libraries:
```
libbrotlienc.a
libclrinterpreter.a
libbrotlidec.a
libcoreclr_static.a
libbrotlicommon.a
libSystem.Security.Cryptography.Native.Apple.a
libSystem.Net.Security.Native.a
libSystem.IO.Compression.Native.a
libSystem.Globalization.Native.a
libSystem.Native.a
```